packages/redux/src/orders/actions/factories/helpers/hashFetchUserOrdersQuery.ts

```diff
@@ -0,0 +1,18 @@
+import type { GetUserOrdersQuery } from '@farfetch/blackout-client';
+
+export type HashedFetchUserOrdersQuery = string;
+
+export default function hashFetchUserOrdersQuery(
```

Comment:
Include the `userId` in the hash as well and change the name to `generateUserOrdersRequestHash`.
You will need to add the `userId` internally on the selectors as well, like this:

```js
export const areUserOrdersLoading = (
  state: StoreState,
  query?: GetUserOrdersQuery,
) => {
  const userId = getUser(state)?.id;

  const requestHash = generateUserOrdersRequestHash(userId, query);
  const loaders = getIsLoading(state.orders as OrdersState);

  return loaders[requestHash];
};
```

Reply:
Done 🙌